integration-tests/src/test/java/org/openmrs/module/fhir2/providers/r4/EncounterFhirResourceProviderIntegrationTest.java

```diff
@@ -818,4 +818,21 @@ public void shouldThrow404WhenUpdatingNonExistingVisitAsJson() throws Exception
 		assertThat(response.getContentType(), is(FhirMediaTypes.JSON.toString()));
 		assertThat(response.getContentAsString(), notNullValue());
 	}
+
+	@Test
+	public void shouldSearchForEncountersByPatientIdentifiersOr() throws Exception {
+		String uri = String.format("/Encounter?subject.identifier=101-6&subject.name=Chebaskwony");
+		MockHttpServletResponse response = get(uri).accept(FhirMediaTypes.XML).go();
+
+		Bundle results = readBundleResponse(response);
```

Comment:
This test is now failing and returning a "Multiple values detected for non-repeatable parameter 'subject'. This server is not configured to allow multiple (AND/OR) values for this param" message